eng/Workarounds.props

```diff
@@ -12,6 +12,11 @@
     <_ProjectExtensionsWereImported>true</_ProjectExtensionsWereImported>
     <WixTargetsPath>$(WixInstallPath)\wix2010.targets</WixTargetsPath>
   </PropertyGroup>
+  
+  <PropertyGroup>
+    <!-- Required because the Razor SDK will generate attributes -->
+    <GenerateRazorAssemblyInfo Condition="'$(UsingMicrosoftNETSdkWeb)' == 'true' AND '$(GenerateRazorAssemblyInfo)' == ''">false</GenerateRazorAssemblyInfo>
```

Comment:
I tried adding `AND '$(TargetFrameworkIdentifier)' == '.NETCoreApp'` but it wasn't working. Likely something to do with a variable not being evaluated yet...